### DISCUSSION TITLE
A guide on how to handle mongoDB views using tsed/mongoose

### DISCUSSION THREAD

**Discussion by nurik24** [TypedProject/ts-express-decorators#1410] [Q&A] [answered]
Hello, i am trying to use mongoDB views but having a hard time on implementation. the docs dont have much info on views. Could you please help me out here, an example would be great. So basically I'm treating views as normal collections creating a model of my view, then injecting it to my service.

**Comment by Romakita** [comment, marked as answer]
Hello @nurik24 
Your need is not clear at all. Assumes that I don't know mongoose and mongoDB (let alone mongoose views). So give me an example with pure mongoose code and then I could give you an answer.
I cannot help you based on your issue details ;)

See you
Romain

**Reply by Romakita** [reply]
I saw that: https://github.com/Automattic/mongoose/issues/5694

So if mongoose doesn't support it, or you haven't example with mongoose, I won't support it also ^^. 

See you

**Reply by Romakita** [reply]
@nurik24 have you found the solution ?

**Reply by nurik24** [reply]
First of all, I am extremely sorry I couldn't get back to you.
I decided not to use them(views), it was and still is a crazy busy period of time for me right now. One of the reasons I decided not to use views is that I was trying to use "$lookup" to join the collections, but the performance was extremely bad, since mongoDB is not a relational db.
Thank you a lot for your prompt replies 👍

**Reply by Romakita** [reply]
No worries ;) 
See you
